src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    IsoscelesTrapezoid(Color color, double base1, double base2, double height) {
```

Comment:
The constructor of `IsoscelesTrapezoid` should not contain expressions. It's better to create separate variables before passing them to the constructor for better readability.